### DISCUSSION TITLE
Question about voltage reference of setTCXO command

### DISCUSSION THREAD

**Discussion by tripla-m** [jgromes/RadioLib#513] [Q&A] [answered]
Hi, i'm Michele

I am using an eByte E22-400M30S and have a question about the setTCXO command: What difference does it make to use a voltage of 1.6 volts or other values? 

Thanks

**Comment by jgromes** [comment, marked as answer]
E22-40030S uses a TCXO that requires 1.6V reference voltage - that's just specified by the manufacturer. Howver, the SX126x does not make any assumptions about specific TCXO being used; another crystal by a different manufacturer might require different voltage reference.

**Comment by tripla-m** [comment]
Thanks, I had searched for this value on the ebyte documents for a long time, but without success. 
Another question, does the crystal fail if a higher voltage is applied?

**Reply by jgromes** [reply]
I would say that again, it depends on the exact crystal. At the very least, I would expect incorrect frequency at the output.
